src/css/bundle.scss

```diff
@@ -807,6 +807,12 @@ input[type='text'].display {
             // }
         }
 
+        &.ammo-inline {
+            margin-left: 18px;
```

Comment:
It's not yet a system wide rule by a wide margin, however using none-px values like shadow suggests will allow CSS rules to be scalable with font size.